### PR TITLE
feat(external-api): enhance recordingLinkAvailable to provide ttl info

### PR DESCRIPTION
**Pull request by mihhu**
Added time to live param for the `recordingLinkAvailable` event.

### CODE REVIEW

**Comment by mihhu** [comment]
Jenkins test this please.

File: react/features/recording/functions.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ export async function getRecordingLink(url: string, recordingSessionId: string,
     });
     const json = await res.json();
 
-    return res.ok ? json.url : Promise.reject(json);
+    return res.ok ? json : Promise.reject(json);
```
Comment (horymury) [inline react/features/recording/functions.js:68]: nit: not related to this PR but we could do something like  : if (res.ok)  return json else throw json as we now mix async/await with promise approach.

**Review by horymury — APPROVED** [review]

**Comment by mihhu** [comment]
Please Jenkins test this please.

**Comment by mihhu** [comment]
Please Jenkins test this please.

**Comment by mihhu** [comment]
Please Jenkins test this please!

**Comment by mihhu** [comment]
Please Jenkins test this please!

**Comment by mihhu** [comment]
Please Jenkins test this please!

**Comment by mihhu** [comment]
Please Jenkins test this please!

**Comment by mihhu** [comment]
Please Jenkins test this please!

**Comment by mihhu** [comment]
Please Jenkins test this please!

**Review by saghul — APPROVED** [review]

**Pull request merged**